### ISSUE TITLE
Critical static analisys issues reported by Klocwork

### ISSUE THREAD

**Issue by antoxar** [mwiede/jsch#11]
![image](https://user-images.githubusercontent.com/921451/87797735-66249f80-c84b-11ea-8f09-c73d97c9474b.png)

**Comment by mwiede** [comment]
Yes I am aware of this. I set up Sonarcloud analysis for this repo.
Compare here https://sonarcloud.io/dashboard?id=mwiede_jsch

**Comment by danielgrahl** [comment]
Sonarcloud is currently not analyzing any Java code.

**Comment by mwiede** [comment]
Correct. I deactivated it as it did not work on pr builds.

**Comment by mwiede** [comment]
Sent current code analysis to Sonarcloud. https://sonarcloud.io/project/overview?id=mwiede_jsch

Java Source and Coverage included
